docs/Credentials.rst

```diff
@@ -51,9 +82,10 @@ For users who would like to save their credentials and let other OS configured a
     import synapseclient
     syn = synapseclient.login("username", "password", rememberMe=True)
 
-The application (keychain in Mac) will then prompt you to allow Python to access these credentials. Please choose “Yes” or “OK”.
+The application (keychain in Mac) will then prompt you to allow Python to access these credentials. Please choose "Yes" or "OK".
 
-The second time you login, you will not have to enter your `username` or `password`::
+The second time you login, you will not have to enter your :code:`username` or :code:`password`::
```

Comment:
same comment as above to remove code formatting here.

Reply:
I changed it to:

The second time you login, you will not have to enter any arguments to `login()`

Reply:
sounds good.